whats_fresh/whats_fresh_api/templates/base.html

```diff
@@ -19,7 +20,9 @@
         <div class="header">
             {% block header %}
             {% endblock header%}
-			<h2><a href='/entry'>{% site_title %}</a></h2>
+            <h2><a href='/entry'>{{ 'site_title'|get_fieldname }}</a></h2>
+            <h4>{% firstof 'slogan'|get_fieldname "" %}</h4>
+            <a><img src="{{ 'logo'|get_fieldname }}"></a>
```

Comment:
@tschuy This here does not displays the image, here is what it gives me - https://i.imgur.com/pyPnw5m.png